resources/lang/vi/layout.php

```diff
@@ -20,241 +20,241 @@
         ],
         'store' => [
             '_' => 'store',
-            'checkout-index' => 'checkout',
-            'getListing' => 'listing',
-            'getCart' => 'cart',
+            'checkout-index' => 'thanh toán',
+            'getListing' => 'danh sách',
+            'getCart' => 'giỏ hàng',
 
-            'getCheckout' => 'checkout',
-            'getInvoice' => 'invoice',
-            'products-show' => 'product',
+            'getCheckout' => 'thanh toán',
+            'getInvoice' => 'hóa đơn',
+            'products-show' => 'mặt hàng',
 
-            'new' => 'new',
-            'home' => 'home',
-            'index' => 'home',
-            'thanks' => 'thanks',
+            'new' => 'mới',
+            'home' => 'trang chủ',
+            'index' => 'trang chủ',
+            'thanks' => 'cảm ơn',
         ],
         'admin-forum' => [
             '_' => 'admin::forum',
-            'forum-covers-index' => 'forum covers',
+            'forum-covers-index' => 'bìa diễn đàn',
         ],
         'admin-store' => [
             '_' => 'admin::store',
-            'orders-index' => 'orders',
-            'orders-show' => 'order',
+            'orders-index' => 'đơn hàng',
+            'orders-show' => 'đơn hàng',
         ],
         'admin' => [
             '_' => 'admin',
-            'beatmapsets-covers' => 'beatmapset covers',
-            'root' => 'index',
+            'beatmapsets-covers' => 'bìa beatmapset',
+            'root' => 'mục lục',
             'logs-index' => 'log',
             'beatmapsets' => [
                 '_' => 'beatmapsets',
-                'show' => 'detail',
+                'show' => 'chi tiết',
             ],
         ],
     ],
 
     'footer' => [
         'general' => [
-            '_' => 'General',
-            'home' => 'Home',
+            '_' => 'Chung',
+            'home' => 'Trang Chủ',
             'changelog-index' => 'Changelog',
-            'beatmaps' => 'Beatmap Listings',
-            'download' => 'Download osu!',
+            'beatmaps' => 'Danh Sách Beatmap',
+            'download' => 'Tải Xuống osu!',
             'wiki' => 'Wiki',
         ],
         'help' => [
-            '_' => 'Help & Community',
-            'faq' => 'Frequently Asked Questions',
-            'forum' => 'CommunityForums',
+            '_' => 'Giúp Đỡ & Cộng Đồng',
+            'faq' => 'Những Câu Hỏi Thường Gặp',
+            'forum' => 'Diễn Đàn',
             'livestreams' => 'Live Streams',
-            'report' => 'Report an Issue',
+            'report' => 'Báo Cáo Một Vấn Đề',
         ],
         'support' => [
-            '_' => 'Support osu!',
+            '_' => 'Ủng Hộ osu!',
             'tags' => 'Supporter Tags',
-            'merchandise' => 'Merchandise',
+            'merchandise' => 'Hàng Hóa',
         ],
         'legal' => [
-            '_' => 'Legal & Status',
-            'copyright' => 'Copyright (DMCA)',
+            '_' => 'Tình Trạng Pháp Lý',
```

Comment:
This one is total fine but adding ``&`` to keep the same format, it's up to you btw

Reply:
yeha